risc0/zkvm/src/serde/serializer.rs

```diff
@@ -402,13 +428,18 @@ impl<'a, W: StreamWriter> serde::ser::SerializeStructVariant for &'a mut Seriali
     }
 }
 
+/// A vector of bytes that can be written to with a stream interface
```

Comment:
This is specifically a host-side `StreamWriter` impl, it's not meant to be used in the guest.

Reply:
:+1: Added a remark indicating it's for host use.